pkg/gogocodec/gogocodec.go

```diff
@@ -18,10 +18,11 @@ import (
 )
 
 const (
-	tempoProtoGenPkgPath  = "github.com/grafana/tempo/pkg/tempopb"
-	jaegerProtoGenPkgPath = "github.com/jaegertracing/jaeger/proto-gen"
-	jaegerModelPkgPath    = "github.com/jaegertracing/jaeger/model"
-	otelProtoPkgPath      = "go.opentelemetry.io/collector"
+	frontendProtoGenPkgPath = "github.com/grafana/tempo/modules/frontend"
```

Comment:
is this necessary? i'd prefer not doing this unless we have to.

Reply:
I got some marshaling error in the e2e tests without it.  Which I think makes sense since we are using the gogoproto generator to write the pb.go files.

Reply:
https://github.com/grafana/tempo/runs/5028797385?check_suite_focus=true